FranciscoPerez/react-challenge-project-jan-2023
# Fp 12 add live refresh

## Changes
1. Add live refresh feature to `/view-orders` page by using `useEffect` and `setInterval` functionality.
2. Use clean up function to clear/stop the fetch orders interval.

## Purpose
Current `/view-orders` page displays static list of orders that were fetched when page was loaded.

[Issue #12](https://github.com/Shift3/react-challenge-project-jan-2023/issues/12)

## Approach
Fetch orders at the same interval as the `/api/live-mode` API endpoint by implementing `useEffect` hook and `setInterval` method.

This will refresh the orders on `/view-orders` page as orders are added/deleted in the database.

## Testing Steps
1. Navigate to the `/view-orders` page.
2. Verify orders change without user interaction.

## Learning
I mainly relied on the React documentation for the "useEffect" hook.

[useEffect Documentation](https://react.dev/reference/react/useEffect)

Closes #12
